widget/ui/src/containers/SwapInput/SwapInput.tsx

```diff
@@ -89,13 +108,19 @@ export function SwapInput(props: SwapInputProps) {
                   warningLevel={props.warningLevel}
                 />
               ) : (
-                <ValueTypography hasWarning={!!props.price.error}>
-                  <Typography variant="body" size="medium">
-                    {props.price.usdValue
-                      ? `~$${props.price.usdValue}`
-                      : props.price.error}
-                  </Typography>
-                </ValueTypography>
+                <Tooltip content={props.price.usdValue?.toString()}>
```

Comment:
![Screenshot from 2023-12-13 12-55-33](https://github.com/rango-exchange/rango-client/assets/120931936/71b56a1e-52e1-4dac-9241-695ec7b5604e)
Tooltip isn't in the right place

Reply:
fixed.